src/sql/workbench/api/node/mainThreadModelViewDialog.ts

```diff
@@ -40,7 +40,7 @@ export class MainThreadModelViewDialog implements MainThreadModelViewDialogShape
 
 	public $close(handle: number): Thenable<void> {
 		let dialog = this.getDialog(handle);
-		dialog.close();
+		this._dialogService.closeDialog(dialog);
```

Comment:
we should dispose the model view when closing the dialog

Reply:
Good call, done now